include/tvm/ir/attrs.h

```diff
@@ -475,6 +475,8 @@ class AttrInitVisitor {
     } else {
       opt.value_missing_ = true;
     }
+#pragma GCC diagnostic ignored "-Wpragmas"
```

Comment:
For compilers except gcc, it may cause pragma warnings either. So, if we add this kind of pragmas , it may be better to add conditional macro to seperate from other compilers. And in llvm and msvc, there may be other pragma instructions simlilar to gcc ?

Reply:
Personally I prefer to avoid adding pragmas for different compilers  for it seems be too tricky. I vote for the 2nd way.

Reply:
* Restrict pragma to GCC only.